app/services/waste_carriers_engine/order_copy_cards_completion_service.rb

```diff
@@ -31,7 +31,7 @@ def update_registration
 
       # Log the order items only if payment is complete.
       if !@transient_registration.unpaid_balance?
-        OrderItemLog.create_from_registration(registration)
+        OrderItemLog.create_from_registration(registration, Time.now)
```

Comment:
We had some issues recently with Times/Zones. Could we use `Time.current` instead? https://apidock.com/rails/v5.2.3/DateTime/current/class